itest/assets_test.go

```diff
@@ -199,6 +199,7 @@ func mintAssetUnconfirmed(t *harnessTest, tapd *tapdHarness,
 	unconfirmedAssets := groupAssetsByName(listRespUnconfirmed.Assets)
 	for _, assetRequest := range assetRequests {
 		metaHash := (&proof.MetaReveal{
+			Type: proof.MetaOpaque,
```

Comment:
IIUC, this was the existing default value, so what does this change add?

Reply:
No, the `proof.MetaOpaque` is value `1`. But on the RPC it's `0`. So we didn't have it translated properly between native and RPC types.